src/vs/workbench/api/test/common/extensionHostMain.test.ts

```diff
@@ -23,6 +24,10 @@ import { ProxyIdentifier, Proxied } from 'vs/workbench/services/extensions/commo
 
 suite('ExtensionHostMain#ErrorHandler - Wrapping prepareStackTrace can cause slowdown and eventual stack overflow #184926 ', function () {
 
+	if (isFirefox || isSafari) {
+		return;
+	}
```

Comment:
If you use `(isFirefox || isSafari ? suite.skip : suite)` they will show up as pending when running the tests, giving a stronger indicator that the test didn't actually pass on that platform